spec/routing/carts_routing_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe Api::V1::CartsController, type: :routing do
```

Comment:
we usually don't add this kind of test but it's ok if you want to keep it

Reply:
I like this section for the routes since it can help us warn of changes that must be considered in other areas of the application.